src/test/java/com/softserveinc/dokazovi/service/impl/PostServiceImplTest.java

```diff
@@ -1278,6 +1279,79 @@ void updatePostById_WhenExists_isOk_DoctorRole() {
 		Assertions.assertThat(postService.updatePostById(userPrincipal, dto)).isTrue();
 	}
 
+	@Test
+	void updatePostById_WhenPostStatus_NotFount_ThrowException() {
```

Comment:
`NotFount` it appears to be mistyping here. It should be `NotFound`

Reply:
Fixed.